test/activity_test.rb

```diff
@@ -0,0 +1,59 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/activity'
+require 'pry'
+
+class ActivityTest < Minitest::Test
+
+  def test_it_exists
+    activity = Activity.new("Brunch")
+    assert_instance_of Activity, activity
+  end
+
+  def test_it_has_name
+    activity = Activity.new("Brunch")
+    assert_equal "Brunch", activity.name
+  end
+
+  def test_participants_is_empty_hash_by_default
+    activity = Activity.new("Brunch")
+    assert_equal ({}), activity.participants
+  end
+
+  def test_it_can_add_participants
+    activity = Activity.new("Brunch")
+    activity.add_participant("Maria", 20)
+    activity.add_participant("Luther", 40)
+    assert_equal ({"Maria" => 20, "Luther" => 40}), activity.participants
+  end
+
+  def test_it_can_get_total_cost
+    activity = Activity.new("Brunch")
+    activity.add_participant("Maria", 20)
+    assert_equal 20, activity.total_cost
+    activity.add_participant("Luther", 40)
+    assert_equal 60, activity.total_cost
+  end
+
+  def test_it_can_split_the_cost
+    activity = Activity.new("Brunch")
+    activity.add_participant("Maria", 20)
+    activity.add_participant("Luther", 40)
+    assert_equal 30, activity.split
+  end
+
+  def test_it_can_calculate_owed_ammount_for_each_participant
+    activity = Activity.new("Brunch")
+    activity.add_participant("Maria", 20)
+    activity.add_participant("Luther", 40)
+    assert_equal ({"Maria" => 10, "Luther" => -10}), activity.owed
+  end
+
+  def test_it_can_print_summary
```

Comment:
`summary` is a method on Reunion, not on Activity.  Did you write this test before or after implementing the summary method?